guides/concepts/transactions.md

```diff
@@ -50,40 +69,127 @@ It's of note that each signer signs the hash of the transaction object in additi
-Stellar-core nominates the transaction set it has collected. SCP resolves the differences between the various transaction sets proposed and decides on the one transaction set that the network will apply.
-
-6. **Application**: Once SCP agrees on a particular transaction set, that set is applied to the ledger. At this point, a fee is taken from the source account for every transaction in that set. Operations are attempted in the order they occur in the transaction. If any operation fails, the whole transaction fails, and the effects of previous operations in that transaction are rolled back. After all the transactions in the set are applied, a new ledger is created and the process starts over.
+## Validity of a Transaction
+
+To determine if a transaction is valid, many checks take place over the course of the transaction's
+lifecycle. The following conditions determine whether a transaction is valid:
+
+* **Source Account** — The source account must exist on the ledger.
+* **Fee** — The fee must be greater than or equal to the network minimum fee for the number of
+  operations submitted as part of the transaction. Note that this does not guarantee that the
+  transaction will be applied; it only guarantees that it is valid. In addition, the source account
+  must be able to pay the fee specified. In the case where multiple transactions are submitted but
+  only a subset of them can be paid for, they are checked for validity in order of sequence number.
+* **Sequence Number** — For the transaction to be valid, the sequence number
+  must be 1 greater than the sequence number stored in the source account [account
+  entry](./accounts.md) _when the transaction is applied_. This means when checking the validity of
+  multiple transactions with the same source account in a candidate transaction set, they must all
+  be valid transactions and their sequence numbers must be offset by 1 from each other. When it
+  comes to apply time, they are ordered and applied according to their sequence number.
+  * For example, if your source account's sequence number is 5 and you submit 3 transactions, all
+    transactions must be considered valid and their sequence numbers must be 6, 7, and 8 in order
+    for any of them to make it into a candidate transaction set.
+* **List of Operations** — Each operation must pass all of the [validity checks for an
+  operation](./operations.md#validity-of-an-operation).
+* **List of Signatures** — In addition to meeting the signature requirements of each operation in the
+  transaction, the following requirements must be met for the transaction:
+  * Was the appropriate network passphrase part of the transaction hash that was signed by each of
```

Comment:
same than with operations : list of requirements should be assertions, not questions